crates/holochain_sqlite/src/sql/cell/fetch_op_hashes_wrap.sql

```diff
@@ -0,0 +1,17 @@
+SELECT
+    hash
+FROM
+    DHtOp
+WHERE
+    DhtOp.authored_timestamp_ms >= :from
+    AND DhtOp.authored_timestamp_ms < :to
+    AND (
+        (
+            storage_center_loc >= :storage_start_1
+            AND storage_center_loc <= :storage_end_1
+        )
+        OR (
+            storage_center_loc >= :storage_start_2
+            AND storage_center_loc <= :storage_end_2
+        )
```

Comment:
I don't know if this `OR` breaks the indexes or not... but another option here would be to UNION ALL two sub selects for the two different ranges.

Reply:
I address this in the other PR. Note that for times where you have a large full sync query this will basically result in scan the table twice but I guess that's the trade off and we are hoping our shards don't get too large I guess. Although I don't see how we can limit the size of the shards as it's dependent on the amount of data on the dht vs the num of peers. In cases where we have very high data density these indexes actually slow things down as it's faster to full table scan when you need to check everything anyway.